src/frontend/src/utils/work-package.utils.ts

```diff
@@ -15,8 +15,34 @@ export const getTitleFromFormType = (formType: WPFormType, wbsElement: WbsElemen
 /**
  * Given a list of work packages, return the work packages that are overdue.
  * @param wpList a list of work packages.
- * @returns a list of work packages that are overdue.
+ * @returns a sub-list of work packages that are not complete, and have end dates before the current date.
  */
 export const getOverdueWorkPackages = (wpList: WorkPackage[]): WorkPackage[] => {
-  return wpList.filter((wp) => wp.status !== WbsElementStatus.Complete && wp.endDate < new Date());
+  return wpList.filter((wp) => wp.status !== WbsElementStatus.Complete && new Date(wp.endDate) <= new Date());
+};
+
+/**
+ * Given a list of work packages, return the work packages that are upcoming.
+ * @param wpList a list of work packages.
+ * @returns a sub-list of work packages that are active and have a start date within the next 2 weeks.
+ */
+export const getUpcomingWorkPackages = (wpList: WorkPackage[]): WorkPackage[] => {
+  return wpList.filter(
+    (wp) =>
+      wp.status !== WbsElementStatus.Complete &&
+      new Date(wp.startDate) <= new Date(new Date().getTime() + 14 * 24 * 60 * 60 * 1000) &&
```

Comment:
Can you add a comment explaining the math here

Reply:
Changed it to use the date util function so no math needed anymore